### PR TITLE
EAS storybook scripts & development build

### PR DESCRIPTION
**Pull request by seanim0920**
Added a script for creating a development build of our app on eas. Added helper scripts for running storybook & changing api configs. Needed to remove some broken imports to get storybook running again.
Removed unused npm scripts
* MAPS_API_KEY, EXPO_PROJECT_OWNER and EXPO_PROJECT_ID need to be added in both eas secrets and .env.infra

### CODE REVIEW

File: app.config.js [hunk 1]
```diff
@@ -2,7 +2,8 @@ const dotenv = require("dotenv")
 
 dotenv.config({ path: ".env.infra" })
 
-const { MAPS_API_KEY, EXPO_PROJECT_ID, EXPO_PROJECT_OWNER } = process.env
+const { MAPS_API_KEY, EXPO_PROJECT_ID, EXPO_PROJECT_OWNER, BUILD_TYPE } =
```
Comment (seanim0920) [inline app.config.js:5]: Edit bundle identifier for ios

**Review by seanim0920 — COMMENTED** [review]
Add script for removing .storybook folder in prod buld

File: package.json [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
-    "prepare": "husky install",
     "staged-lint": "lint-staged",
-    "dev": "expo start --dev-client",
-    "eas-build-post-install": "node easbuild.mjs create \"EAS Build\"",
-    "eas-build-on-success": "node easbuild.mjs success \"EAS Build\"",
-    "eas-build-on-error": "node easbuild.mjs failure \"EAS Build\"",
-    "eas-build-on-cancel": "node easbuild.mjs cancelled \"EAS Build\"",
-    "eas-build-local": "eas build --profile development --platform android",
-    "eas-build-storybook-local": "cd .storybook && eas build --profile development --platform android"
+    "eas_build_local": "eas build --profile development --platform android",
```
Comment (seanim0920) [inline package.json:11]: remove android

File: development-google-services.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
+{
```
Comment (seanim0920) [inline development-google-services.json:1]: Remove, not needed atm

**Review by mhayes853 — APPROVED** [review]

**Pull request merged**